### DISCUSSION TITLE
大家的本地开发环境和工具用的是什么?

### DISCUSSION THREAD

**Discussion by wei98k** [go-kratos/kratos#738] [General]
1. mac+vscode
2. mac+goland
3. windows7+vbox+linux+goland(OR vscode)
4. windows10+wsl+goland(OR vscode)
5. windows10+docker+goland(OR vcode)

**Comment by wei98k** [comment]
mac+goland

**Comment by wei98k** [comment]
windows10+docker+goland(OR vcode)

**Comment by sdttttt** [comment]
windows10 + vscode

**Comment by sxlvalue** [comment]
windows11+wsl2+DockerDesktop+goland
